translationCoreApps/ScripturePane
# Reformat with flex and fix bugs

#### This pull request addresses:

This PR directly fixes two minor issues, #1597 and #1520 as well as revamps the entire Scripture Pane using flex in preparation for issue #1168.  Lots of outdated code was also removed.



#### How to test this pull request:

Open a project and make sure all elements of the Scripture Pane still look good.  (There were lots of changes to the way styling and formatting are done, but should not be noticeable to the user).  Make sure the expanded pane has headings now (with the full language name).


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/48)
